.github/workflows/playground-petclinic-maven-sb-2.yaml

```
@@ -0,0 +1,61 @@
+      NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+      NEXUS_HOST_PORT: ${{ secrets.NEXUS_HOST_PORT }}
+
+      YANDEX_CLOUD_OAUTH2_TOKEN: ${{ secrets.YANDEX_CLOUD_OAUTH2_TOKEN }}
+      CR_URL: ${{ vars.CR_URL }}
+      SANDBOX_KUBE_CONFIG: ${{ secrets.SANDBOX_KUBE_CONFIG }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v6
+
+      - name: Loading Secrets
+        uses: ./.github/actions/bitwarden
```

actionlint (1.7.12)

[error] description is required in metadata of "Bitwarden Credentials Integration" action at "/home/jailuser/git/.github/actions/bitwarden/action.yaml"

(action)